src/test/kotlin/no/nav/helsearbeidsgiver/inntekt/MockInntektKlient.kt

```diff
@@ -26,6 +28,6 @@ fun mockInntektKlient(
 
     return mockStatic(::createHttpClient) {
         every { createHttpClient() } returns mockHttpClient
-        InntektKlient("url") { "mock access token" }
+        InntektKlient("baseUrl", LocalCache.Config(Duration.ZERO, 1)) { "mock access token" }
```

Comment:
Named arguments på denna?
```suggestion
        InntektKlient("baseUrl", LocalCache.Config(entryDuration = Duration.ZERO, maxEntries = 1)) { "mock access token" }
```